pyproject.toml

```diff
@@ -22,36 +22,38 @@
 
 [tool.poetry]
 name = "krux"
-version = "22.08.2"
+version = "23.07.beta15"
```

Comment:
needs to be updated to 23.08 (or 23.09 if that ends up happening, ha)

Reply:
I've done this, on PR now